Makefile

```diff
@@ -8,11 +8,10 @@ WORKER_PID=airflow-worker.pid
 FLOWER_PID=airflow-flower.pid
 
 init:
-	pyenv global $(PYTHON_VERSION)
-	pyenv install ${PYTHON_VERSION}
-	pyenv virtualenv ${PYTHON_VERSION} workflows
-	pyenv activate workflows
+	poetry env use 3.11
+	poetry shell
```

Comment:
why do we need `poetry shell`? if we run poetry shell then there is no need to run `poetry run`